### PR TITLE
protonmail-bridge: init at 1.0.5-1

### PR DESCRIPTION
**Pull request by lightdiscord**
###### Motivation for this change

Add a new package to nixpkgs.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/networking/protonmail-bridge/default.nix [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+  desktopItem = makeDesktopItem {
+    name = "Desktop-Bridge";
+    exec = "Desktop-Bridge";
+    icon = "desktop-bridge";
+    comment = stdenv.lib.replaceStrings ["\n"] [" "] meta.description;
+    desktopName = "ProtonMail Bridge";
+    genericName = "ProtonMail Bridge for Linux";
+    categories = "Utility;Security;Network;Email";
+  };
```
Comment (Infinisil) [inline pkgs/applications/networking/protonmail-bridge/default.nix]: Minor thing: Can you move the desktopItem declaration into the `let in`? It's not actually needed as an attribute to the derivation, not sure why a lot of derivations do it nonetheless.

Other than that this PR looks perfect :)

**Comment by LightDiscord** [comment]
cc @xeji

**Review by xeji — COMMENTED** [review]
Please modify according to nixpkgs [conventions for qt applications](https://nixos.org/nixpkgs/manual/#ssec-qt-applications).

**Comment by LightDiscord** [comment]
It should be ok.

**Comment by xeji** [comment]
@GrahamcOfBorg build protonmail-bridge

File: pkgs/applications/networking/protonmail-bridge/default.nix (hunk 1 above)
Comment (Infinisil) [reply pkgs/applications/networking/protonmail-bridge/default.nix]: Oh I see, didn't notice, well it's fine either way

**Pull request merged**